### ISSUE TITLE
modify the code applicable to Ouster64 data

### ISSUE THREAD

**Issue by G12311231** [Natsu-Akatsuki/RangeNetTrt8#19]
Hello，
Thank you very much for the open source！

I tried to directly use the pre-trained model provided by lidar-bonnetal to segment the data collected by ouster64. The segmentation results were very strange. How can I modify the code to make it applicable to ouster64 data?

**Comment by Natsu-Akatsuki** [comment]
I’m afraid not. The pretrained model is specifically tailored for the Velodyne-64.

**Comment by G12311231** [comment]
Thank you for your answer! I want to know if it is because the rangenet++ method converts a three-dimensional point cloud into a two-dimensional one based on a distance image, so different types of sensors cannot use the model trained by the author based on vlp-64.
And If I use the same sensor vlp-64 to collect data on campus, would like to know if this dataset can use the parameters of the pre-trained model?

**Comment by Natsu-Akatsuki** [comment]
1. Yes. This is one reason. It is welcome to implement the projection method (3D -> 2D) to adapt for Ouster-64 and it may work. Another concern is the domain gap between the two LiDAR, which may lead to decrease in accuracy.
2. Look so good. However, as I have not personally conducted the tests, I cannot provide you with a definitive answer with 100% confidence.

**Comment by G12311231** [comment]
> 1. Yes. This is one reason. It is welcome to implement the projection method (3D -> 2D) to adapt for Ouster-64 and it may work. Another concern is the domain gap between the two LiDAR, which may lead to decrease in accuracy.
> 2. Look so good. However, as I have not personally conducted the tests, I cannot provide you with a definitive answer with 100% confidence.

I understand, and thank you very much for your reply!
